doc/sg/c_cpp_coding_style.md

```diff
@@ -144,7 +156,7 @@ You must not dereference a symbol that has non-address value.
 ***It is recommended to document public functions, classes, methods, and data structures in the header file with a Doxygen-style comment.***
 
 The first line of the comment is the summary, followed by a new line, and an optional longer description.
-Input arguments and return arguments can be documented with `@param` and `@return` if they are not self-explanatory from the name.
+Input arguments and return arguments can be documented with `@param` and `@return` if they are not self-explanatory from the name. Output arguments should be documented with `@param[out]`.
```

Comment:
Question: 'can' be documented or 'should'?
Can is weak, whereas should is more explicit. Might be worth checking the canonical reference to see if it is mentioned.

Reply:
Should is better